internal/metrics/metrics.go

```diff
@@ -113,6 +114,12 @@ func NewPrometheusReporter(reg prometheus.Registerer, logger Logger) (*Prometheu
 			Help:      "Version info of strong_duckling",
 		}, []string{"version"}),
 		tcpChecker: &tcpChecker{
+			checks: prometheus.NewCounterVec(prometheus.CounterOpts{
+				Namespace: namespace,
+				Subsystem: subSystemTcpChecker,
+				Name:      "checked_total",
+				Help:      "Total number of times the connection has been checked",
+			}, []string{"name", "address", "port"}),
```

Comment:
would be really nice with a `successful` label here. That would easily tell us if how many checks that's successful and would still allow all checks with a `sum by (name)`
```suggestion
			}, []string{"name", "address", "port", "check_status"}),
```